pkg/controllermanager/controller/shoot/shoot_control.go

```diff
@@ -217,7 +228,7 @@ func (c *Controller) updateShootStatusPending(shoot *gardenv1beta1.Shoot, messag
 		func(shoot *gardenv1beta1.Shoot) (*gardenv1beta1.Shoot, error) {
 			shoot.Status.LastOperation = &gardencorev1alpha1.LastOperation{
 				Type:           gardencorev1alpha1helper.ComputeOperationType(shoot.ObjectMeta, shoot.Status.LastOperation),
-				State:          gardencorev1alpha1.LastOperationStateProcessing,
+				State:          gardencorev1alpha1.LastOperationStatePending,
```

Comment:
Why has this been changed? Can we revert it? We don't have a "pending" state anywhere

Reply:
ok, I was talking about this with @zanetworker  - either the name of the function should be  updateShootStatusToProcessing or we update the State to LastOperationStatePending. 
For the scheduler the Pending state would actually fit in my opinion - it waits for the scheduler to act.
If the shoot should just not be reconciled Processing fits better.
What do you think @rfranzke ?

Reply:
Then change the function name. Back then, when this was introduced I talked to the dashboard colleagues and we decided that we don't want to have another "pending" state.